numpy/core/src/multiarray/compiled_base.c

```diff
@@ -1703,6 +1703,9 @@ pack_bits(PyObject *input, int axis, char order)
 static PyObject *
 unpack_bits(PyObject *input, int axis, PyObject *count_obj, char order)
 {
+    static int unpack_init = 0;
+    static npy_uint64 unpack_lookup_l[256];
+    static npy_uint64 unpack_lookup_b[256];
```

Comment:
can you remove the endianness dependence by making the definition `npy_uint8[256][8]`? Does that have any effect on performance?

Reply:
yes that could be done, but lookup table construction then becomes two looks instead of one. Might be less readable.

Reply:
though removing endian mentions from the code might be worth it
I'll update.

Reply:
done